tests/functional/karate_tests/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
is this addition license file required? if not remove it; if so, should it move it to /licenses?